cmd/distrogen/distrogen.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"io/fs"
+	"os"
+)
+
+type Distrogen struct {
```

Comment:
Since this is nominally a part of a "project" (this is essentially a "distrogen project folder") is it possible to fold this functionality into the `ProjectGenerator`?